### ISSUE TITLE
[Question] Future plans

### ISSUE THREAD

**Issue by Redferne** [auchter/haaska#93]
Been enjoying Haaska now for a few months and it works perfectly. However, I think we are missing out on the new consolidated V3 API. We could get some really cool stuff going with Media Control, TemperatureSensor, Camera,  LightColor etc. Does anyone has some insight as what's going on within the HA community? I believe the biggest feature of Haaska is that it is zero configuration. Abit of pain to setup, but once done,  you won't need to touch the Lambda again. And "all" your pre-configured HA devices just works, without the need to manually re-map them in the Amazon Alexa API. Also you dont need to invoke the skill by its magic name, this is close to perfect integration. Contrary to the native intent interface for Alexa in HA.

Any ideas?

**Comment by sumitrajancse** [comment]
I would love to have this working on hassio. I guess there should be one for hassio

**Comment by keatontaylor** [comment]
I have heard mention of a cloud API for HA that will work with the v3 smart home skills API provided by alexa. The issue is that it still looks far off and the HA code currently is not very feature rich like haaska.

**Comment by maddox** [comment]
> I would love to have this working on hassio. I guess there should be one for hassio

This is a Smart Home Skill. It has to be hosted on Amazon Lambda and can not be hosted locally.

**Comment by keatontaylor** [comment]
@maddox, yes but it seems the owner of HA is brewing up a cloud based adapter to allow for a home assistant skill for alexa. Similar to how openhab does it.

**Comment by master-kenobi** [comment]
Except that it's not going to be free. I really hope Haaska will continue to be developed. It's not working right now for me.

**Comment by sumitrajancse** [comment]
Well... I have hassio installed on rpi3 and get the amazon Alexa working using emulated_hue component. Please check the emulated_hue component page on home assistant website . It is very simple and works pretty well.
In case you want hassio and amazon Alexa to work in simple way and have trouble configuring it , I can guide and share my configuration setting. 
The new cloud component will allow any entity to be control via Amazon Alexa, but in case you want to control lights and scenes you can use emulated_hue component.

**Comment by cbrherms** [comment]
I have haaska working with hassio. It's not impossible, just the documentation could be a little better with the changes made to aws development console. 

Emulated_hue works in a totally different way to haaska. They're pretty incompatible. 
Saying to change to that is the same as saying to move to openhab instead of HA to pair with your Alexa.

**Comment by sumitrajancse** [comment]
Yes... I agree.. 
But at the end of day , we want to control lights via Alexa (voice command) and emulated_hue very well serve that purpose.
So there is no harm in using the emulated_hue component.

**Comment by cbrherms** [comment]
For lights, yes, for switches, yet, but for setting other things like thermostats or many other applications that don't have a simple on/off state then no.

**Comment by master-kenobi** [comment]
emulated_hue expose every devices as lights making it other devices such as fans, TV and AC unable to support the true potential of Alexa-enabled Smart Home Group in Alexa app.
